services/web/pkg/config/options.go

```diff
@@ -43,13 +43,18 @@ type FeedbackLink struct {
 // Sidebar are the sidebar option
 type Sidebar struct {
 	Shares SidebarShares `json:"shares" yaml:"shares"`
+	Exif   SidebarExif   `json:"exif" yaml:"exif"`
 }
 
 // SidebarShares are the options for the shares sidebar
 type SidebarShares struct {
 	ShowAllOnLoad bool `json:"showAllOnLoad" yaml:"showAllOnLoad" env:"WEB_OPTION_SIDEBAR_SHARES_SHOW_ALL_ON_LOAD" desc:"Sets the list of the (link) shares list in the sidebar to be initially expanded. Default is a collapsed state, only showing the first three shares." introductionVersion:"pre5.0"`
 }
 
+type SidebarExif struct {
+	ShowLocation bool `json:"showLocation" yaml:"showLocation" env:"WEB_OPTION_SIDEBAR_EXIF_SHOW_LOCATION" desc:"Shows or hides the location data in the EXIF sidebar panel." introductionVersion:"6.0"`
```

Comment:
How does `Shows or hides...` correspond to the values to be set and any default?

Reply:
I added the default and simplified the first part a bit. The value is a `bool`, I won't add details what `true` or `false` does because that's already reflected by the var name.